src/main/kotlin/org/rust/lang/core/psi/util/RustPsiExtensions.kt

```diff
@@ -22,16 +22,17 @@ inline fun <reified T : PsiElement> PsiElement.parentOfType(strict: Boolean = tr
 inline fun <reified T : PsiElement> PsiElement.childOfType(strict: Boolean = true): T? =
     PsiTreeUtil.findChildOfType(this, T::class.java, strict)
 
-fun PsiElement?.getNextNonPhantomSibling(): PsiElement? =
```

Comment:
This method was dead code (and so the `PHANTOM_TOKEN_SET` used only in it), so I have allowed myself to just replace it with two sleeker alternatives.